server/monitor/pom.xml

```diff
@@ -118,6 +118,10 @@
       <groupId>org.glassfish.jersey.containers</groupId>
       <artifactId>jersey-container-servlet-core</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.glassfish.jersey.ext</groupId>
+      <artifactId>jersey-bean-validation</artifactId>
```

Comment:
That should be runtime scope, since it's not needed at compile. Runtime will be included in test.